### PR TITLE
Mark hidden questions 14260

### PR DESCRIPTION
**Pull request by mbischof**
Fixed issue # : 
New feature # : 14260 - mark hidden questions
Changed feature # :
Dev: 
Dev: 

in assets/packages/adminpanel - folder

- removed package-lock.json
- npm install
- gulp

### CODE REVIEW

File: application/models/Question.php [hunk 1]
```diff
@@ -1132,4 +1132,17 @@ public function insertRecords($data)
         Yii::log(\CVarDumper::dumpAsString($oRecord->getErrors()), 'warning', 'application.models.Question.insertRecords');
     }
 
+    public function getQuestionAttribute($name, $default = null)
```
Comment (Shnoulle) [inline application/models/Question.php]: Good function but

1. Need i18n for future
2. default (if not set here) must come from attribute definition

Comment (c-schmitz) [reply application/models/Question.php]: Please use getQuestionAttributes() from model QuestionAttribute instead.

**Comment by dominikvitt** [comment]
This is a great feature.
Can you close this one and create a new pull request in develop branch?
All new features go there.

Also issues mentioned above should be solved.
As soon as you are finished, I'll review it.

File: assets/packages/adminpanel/src/components/subcomponents/_questionsgroups.vue [hunk 2]
```diff
@@ -231,99 +231,104 @@ export default {
...
-                                    </span> 
+                                    :href="question.link"
+                                    class="col-12 pjax question-question-list-item-link display-as-container"
+                                    @click.stop.prevent="openQuestion(question)"
+                                >
+                                    <span v-if="question.hidden == 0" class="question_text_ellipsize" :style="{ width: itemWidth }">
+                                        [{{question.title}}] &rsaquo; {{ question.question_flat }}
+                                    </span>
+
+                                    <span v-if="question.hidden == 1" class="question_text_ellipsize question-hidden" :style="{ width: itemWidth }">
```
Comment (lacrioque) [inline assets/packages/adminpanel/src/components/subcomponents/_questionsgroups.vue]: This is rather verbose a simple :class="question.hidden == 1 ? 'question-hidden' : '' " would suffice

**Comment by mbischof** [comment]
https://github.com/LimeSurvey/LimeSurvey/pull/1283

**Pull request closed without merging**